tests/idl/rapidjson_generator/RapidJsonTest.cpp

```diff
@@ -0,0 +1,205 @@
+#include "gtest/gtest.h"
```

Comment:
This changes creates a new "category" of tests in the source tree.  Is that needed or can it be part of `tests/DCPS/Compiler`?